app/scripts/app-controller.js

```diff
@@ -1,7 +1,17 @@
 'use strict';
 
 module.exports = [
-  '$scope',
-  function AppController($scope) {
+  '$scope', '$http', 'API_URL',
+  function AppController($scope, $http, API_URL) {
     $scope.greeting = 'Hello from scope';
+
+    $http.get(API_URL+'article-states/')
```

Comment:
This should have been put into service which is called on `resolve` callback: http://www.jvandemo.com/how-to-resolve-angularjs-resources-with-ui-router/ (Ignore them using `$resource`, pay attention to configuration of `$stateProvider` and `resolve` callback